gtable/table.py

```diff
@@ -342,5 +342,18 @@ def __setattr__(self, key, value):
                 else:
                     self.add_column(key, value)
 
+    def __getstate__(self):
+        index = self.index
```

Comment:
You have to use self.index.copy() to avoid collisions with existing objects

Reply:
Also, please add a test of this new functionality that refers to #4 (this pull request) in the docstrings, and run the tests to check that everything is correct. When you are done run git commit --amend to add these changes to your previous commit and push.